tests/common/aws_test_runner.c

```diff
@@ -98,38 +98,6 @@ static void RunTestSuites( void )
         RUN_TEST_GROUP( GGD_Helper_System );
     #endif
 
-    #if ( testrunnerFULL_SHADOW_ENABLED == 1 )
-        RUN_TEST_GROUP( Full_Shadow_Unit );
-        RUN_TEST_GROUP( Full_Shadow );
-    #endif
-
-    #if ( testrunnerFULL_SHADOWv4_ENABLED == 1 )
-        RUN_TEST_GROUP( Shadow_Unit_Parser );
-        RUN_TEST_GROUP( Shadow_Unit_API );
-        RUN_TEST_GROUP( Shadow_System );
-    #endif /* if ( testrunnerFULL_SHADOWv4_ENABLED == 1 ) */
-
-    #if ( testrunnerFULL_MQTTv4_ENABLED == 1 )
-        RUN_TEST_GROUP( MQTT_Unit_Validate );
-        RUN_TEST_GROUP( MQTT_Unit_Subscription );
-        RUN_TEST_GROUP( MQTT_Unit_Receive );
-        RUN_TEST_GROUP( MQTT_Unit_API );
-        RUN_TEST_GROUP( MQTT_Unit_Metrics );
-        RUN_TEST_GROUP( MQTT_System );
-    #endif /* if ( testrunnerFULL_MQTTv4_ENABLED == 1 ) */
-
-    #if ( testrunnerFULL_MQTT_STRESS_TEST_ENABLED == 1 )
-        RUN_TEST_GROUP( Full_MQTT_Agent_Stress_Tests );
```

Comment:
why remove this test?

Reply:
This is from a very old library that was deprecated two years ago